packages/core/src/__tests__/__e2e__/progress/LinearProgress.cy.tsx

```diff
@@ -15,7 +15,14 @@ describe("GIVEN a LinearProgress", () => {
     cy.mount(<Default min={20} max={40} value={35} />);
     cy.findByRole("progressbar").should("have.attr", "aria-valuemax", "40");
     cy.findByRole("progressbar").should("have.attr", "aria-valuemin", "20");
-    cy.findByRole("progressbar").contains("75 %");
+    cy.findByRole("progressbar").contains("75%");
```

Comment:
Is this a design update? If so, need to mention in changeset.

Reply:
@navkaur76 I think in the designs we didn't have a space between the number and the "%" in the label, should we double check?

Reply:
Nav will be OOO this week. Let's revert back and update later if needed.